inline_scan.sh

```diff
@@ -469,14 +472,31 @@ start_analysis() {
 		printf '\n%s\n' "***END SERVICE RESPONSE****" >&2
 	    fi
 	    exit 1
-	else
-	    if [ -f /tmp/sysdig/sysdig_output.log ]; then
-		cat /tmp/sysdig/sysdig_output.log
-	    fi
+	fi
+
+    FULLTAG="${SCAN_IMAGES[0]}"
+    check_status_with_digest
+}
+
+check_status_with_digest() {
+    RETRIES=100
+    # Then fetching the result of each scanned digest
+    for ((i=0;  i<${RETRIES}; i++)); do
+        status=$(curl -s -k  --header "Content-Type: application/json" -H "Authorization: Bearer ${SYSDIG_API_TOKEN}" "${SYSDIG_SCANNING_URL}/images/${IMAGE_DIGEST}/checkSummary?tag=$FULLTAG" | grep "status" | awk '{print $2}')
```

Comment:
This command will keep the last comma and quotes in `$status`, making the check later to return always fail:

```
➜  secure-inline-scan git:(add-get-json-inline-scan) ✗ curl -s -k --header "Content-Type: application/json" -H "Authorization: Bearer c116da15-4d54-4f05-90e1-19633c389394" https://secure-staging.sysdig.com/api/scanning/v1/images/sha256:72c42ed48c3a2db31b7dafe17d275b634664a708d901ec9fd57b1529280f01fb/checkSummary\?tag\=alpine | grep "status" | awk '{ print $2 }'
"pass",
```

The simplest I could come up with to fix this is:

```
...| grep "status" | cut -d : -f 2 | awk -F\" '{ print $2 }'
```
 maybe there's something simpler.